public/scss/components/_buttons.scss

```diff
@@ -71,22 +71,36 @@ a[role='button'] {
 }
 
 .back-button {
-  margin-bottom: $space-md;
-  padding: 0;
-  text-decoration: none;
   display: inline-block;
-  border-bottom: 1px solid;
-  font-size: 0.9em;
```

Comment:
Hey Nick, why did you delete these? The border is too chunky on a small screen.

I am less worried about if the link text itself is smaller (it was before to match the "french" link text).